The Straits Times.js

```diff
@@ -105,14 +93,20 @@ function scrape(doc, url) {
 	newItem.complete();
 }
 
-function getMultipleItems(doc, url) {
+function sleep(ms) {
+	return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+async function getMultipleItems(doc, url) {
 	var items = [];
 	var rows;
-	if (url.includes('/search?') && url.includes('searchKey')) {
+	Zotero.debug(url);
+	if (url.includes('/search?') && url.includes('searchkey')) {
+		await sleep(1000);
```

Comment:
@AbeJellinek apparently the third party tool that the site uses for search loads much slower, hence the sleep here. but i am not sure if this is the right way to go about it... Also when I tried to run and update the test, it went into a perpetual 'updating' mode?